source/ReactLoadableSSRAddon.js

```diff
@@ -247,7 +249,7 @@ class ReactLoadableSSRAddon {
       || '';
     this.getEntrypoints(this.stats.entrypoints);
 
-    this.getAssets(this.getMinimalStatsChunks(compilation.chunks));
+    this.getAssets(this.getMinimalStatsChunks(compilation.chunks, compilation.chunkGraph));
```

Comment:
Just in case this isn't present for webpack 4
```suggestion
    this.getAssets(this.getMinimalStatsChunks(compilation.chunks, compilation.chunkGraph || null));
```

Reply:
I don't think chunkGraph is present in webpack4, but not sure to understand why this call is needed? Do we want null instead of undefined?

Reply:
That's true.  Just trying to avoid strict mode madness where possible, but this might not be entirely needed.